tiledb/sm/filesystem/win.cc

```diff
@@ -548,8 +556,9 @@ Status Win::write(
   uint64_t file_offset = file_size_lg_int.QuadPart;
   if (!write_at(file_h, file_offset, buffer, buffer_size).ok()) {
     CloseHandle(file_h);
-    return LOG_STATUS(
-        Status_IOError(std::string("Cannot write to file '") + path));
+    return LOG_STATUS(Status_IOError(
+        std::string("Cannot write to file '") + path + "'" + " file_offset " +
+        std::string(file_offset) + " buffer_size " + std::string(buffer_size)));
```

Comment:
I don't think doing the same with writes is necessary. We got the size from the OS just a few lines above, and the offset is always set to the file's end.